src/components/equations/ItemEffects.test.ts

```diff
@@ -0,0 +1,49 @@
+import { expect } from "chai";
+import { stubBlankGame } from "../../fakes.test";
+import { IPokemon } from "../Battles";
+import { ItemEffects } from "./ItemEffects";
+
+describe("ItemEffects", () => {
+    describe("capturePokemon", () => {
+        it("adds a new Pokemon to your party when your party has less than six Pokemon", (): void => {
+            // Arrange
+            const { fsp } = stubBlankGame();
+            const pokemon: IPokemon = fsp.equations.newPokemon({
+                level: 5,
+                title: "SQUIRTLE".split(""),
+            });
+            const itemEffects = new ItemEffects(fsp);
+
+            // Act
+            itemEffects.capturePokemon(pokemon);
+
+            // Assert
+            expect(fsp.itemsHolder.getItem(fsp.storage.names.pokemonInParty)[0].title).to.be.equal(pokemon.title);
```

Comment:
This asserts the first Pokemon is the new one. You want to assert that the second Pokemon is.